README.md

```diff
@@ -252,37 +254,26 @@ imported with [`go_repository`](#go_repository), will have libraries named
 
 ## Repository rules
 
-### `go_repositories`
+### `go_rules_dependancies`
 
 ``` bzl
-go_repositories(go_version)
+go_rules_dependancies()
 ```
 
 Adds Go-related external dependencies to the WORKSPACE, including the Go
 toolchain and standard library. All the other workspace rules and build rules
 assume that this rule is placed in the WORKSPACE.
+When nested workspaces arrive this will be redundant.
+
+### `go_register_toolchains`
+
+``` bzl
+go_register_toolchains(go_version)
+```
+
+Installs the built in go toolchains.
+If go_version is specified it sets the go version to use by default.
```

Comment:
Installs the Go toolchains. If `go_version` is specified, it sets the SDK version to use (for example, `"1.8.2"`). By default, the latest SDK will be used.

Reply:
Done